classes/migration/upgrade/v3_5_0/I4860_MigratePublicationVersion.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @file classes/migration/upgrade/v3_5_0/I4860_MigratePublicationVersion.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class I4860_MigratePublicationVersion
+ *
+ * @brief Add additional columns for publication versioning and migrate existing data
+ */
+
+namespace PKP\migration\upgrade\v3_5_0;
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+use PKP\migration\Migration;
+use PKP\publication\enums\VersionStage;
+
+class I4860_MigratePublicationVersion extends Migration
```

Comment:
Did you forget the install migration? It needs the new columns too.

Reply:
Added to install/OJSMigration